app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -286,88 +149,157 @@ export const SendBitcoinConfirmationScreen = ({
     update: () => queryTransactions(),
   })
 
-  const pay = async () => {
-    if ((amountless || paymentType === "onchain") && paymentSatAmount === 0) {
+  const handlePaymentReturn = (status, errors) => {
+    if (status === "SUCCESS") {
+      queryWallet(client, "network-only")
+      setStatus(Status.SUCCESS)
+    } else if (status === "PENDING") {
+      setStatus(Status.PENDING)
+    } else {
       setStatus(Status.ERROR)
-      setErrs([{ message: translate("SendBitcoinScreen.noAmount") }])
-      return
+      setErrs(errors)
```

Comment:
I think this traces back to errors returned from the server which won't be localised and probably won't be safe to display to the user.